styles/pages/_product.scss

```diff
@@ -0,0 +1,39 @@
+.product-page {
+  margin-bottom: 50px;
+
+  &__title {
+    margin: 0;
+    text-align: left;
+  }
+
+  &__price {
+    font-weight: bold;
+    font-size: .9rem;
+    margin-bottom: 2rem;
+  }
+
+  &__description {
+    font-weight: 300;
+    line-height: 1.5;
+    margin-bottom: 2rem;
+  }
+
+  &__buy {
+    display: inline-block;
+    width: 100%;
+    max-width: 400px;
+    font-size: 1.8rem;
+    color: $white;
+    background: $accessible-red;
+    padding: 1rem;
+    text-align: center;
+    font-family: 'Timothy';
+    text-decoration: none;
+
+    &:hover,
+    &:active,
+    &:focus {
+      // TODO - what should this be?? 🤔
```

Comment:
@vanitabarrett - thoughts for this?

Reply:
<img width="420" alt="Screenshot 2020-11-10 at 10 51 05" src="https://user-images.githubusercontent.com/29889908/98665215-473ea180-2343-11eb-9710-f0fac8b9c958.png">

Reply:
Done 👍